Sources/MessagingPushAPN/MessagingPushAPN+PushConfigs.swift

```diff
@@ -45,6 +45,6 @@ extension MessagingPushAPN {
     // Swizzled method for `didFailToRegisterForRemoteNotificationsWithError'
     @objc
     func application(_ application: UIApplication, didFailToRegisterForRemoteNotificationsWithError error: Error) {
-        messagingPush.deleteDeviceToken()
+        Self.shared.deleteDeviceToken()
```

Comment:
curious to know why is this only crashing for expo users?

Reply:
This should be crashing for all users but the issue has only been reported by 1 expo user. The use case is rare and only happens if the device is unable to registerFor remote notifications for some reason.